ecosystem/sep-0031.md

```diff
@@ -256,16 +292,63 @@ Content-Type: application/json
+
+Note that the amount delivered to the Receiving Client in this scenario varies depending on the exchange rate used by the Receiving Anchor, which may differ from the rate provided in the `GET /price` response.
+
+```
+POST DIRECT_PAYMENT_SERVER/transactions
+Content-Type: application/json
+
+{
+  "amount": 100,
+  "asset_code": "USD",
+  "asset_issuer": "GDRHDSTZ4PK6VI3WL224XBJFEB6CUXQESTQPXYIB3KGITRLL7XVE4NWV",
+  "destination_asset": "iso4217:BRL",
+  "sender_id": "d2bd1412-e2f6-4047-ad70-a1a2f133b25c",
+  "receiver_id": "137938d4-43a7-4252-a452-842adcee474c",
+  "fields": {
+    "transaction": {
+      "receiver_routing_number": "442928834",
+      "receiver_account_number": "0029483242",
+      "type": "SEPA"
+    }
+  }
+}
+```
+---
+
+The following is an example request body if the Sending Anchor requested a _firm_ quote from [`SEP-38 POST /quote`](sep-0038.md#post-quote) and found the rate acceptable. Because the assets and amounts involved in the transaction are defined in the `POST /quote` request and response, only a `quote_id` attribute is required.
+
+```
+POST DIRECT_PAYMENT_SERVER/transactions
+Content-Type: application/json
+
+{
+  "quote_id": "2bc5b322-5117-413f-869f-e7ca494cb1a4",
+  "sender_id": "d2bd1412-e2f6-4047-ad70-a1a2f133b25c",
+  "receiver_id": "137938d4-43a7-4252-a452-842adcee474c",
+  "fields": {
+    "transaction": {
+      "receiver_routing_number": "442928834",
+      "receiver_account_number": "0029483242",
+      "type": "SEPA"
+    }
+  }
+}
+```
 
 ##### Request Parameters
 
 Name | Type | Description
 -----|-----|------
-`amount` | number | Amount of payment in destination currency.
-`asset_code` | string | Code of the asset the Sending Anchor intends to send. This must match one of the entries listed in the receiving anchor's `GET /info` endpoint.
+`amount` | number | (optional) Amount of the Stellar asset sent to the Receiving Anchor.
+`asset_code` | string | (optional) Code of the asset the Sending Anchor intends to send. This must match one of the entries listed in the receiving anchor's `GET /info` endpoint.
 `asset_issuer` | string | (optional) The issuer of the Stellar asset the Sending Anchor intends to send. If not specified, the asset sent must be issued by the Receiving Anchor.
+`destination_asset` | string | (optional) The off-chain asset the Receiving Anchor will deliver to the Receiving Client. The value must match one of the `asset` values included in a [`SEP-38 GET /prices?sell_asset=stellar:<asset_code>:<asset_issuer>`](sep-0038.md#get-prices) response using [SEP-38 Asset Identification Format](sep-0038.md#asset-identification-format).
+`quote_id` | string | The `id` returned from a `SEP-38 POST /quote` response. If this attribute is specified, none of the attributes listed above should also be included in the request. The Stellar transaction used to send the asset _must_ have a `created_at` timestamp that is earlier than the quote's `expires_at` attribute.
```

Comment:
🤔  Should we be including this `created_at` attribute you're referring to? Or should it actually be `started_at`?

```suggestion
`quote_id` | string | The `id` returned from a `SEP-38 POST /quote` response. If this attribute is specified, none of the attributes listed above should also be included in the request. The Stellar transaction used to send the asset _must_ have a `started_at` timestamp that is earlier than the quote's `expires_at` attribute.
```

Reply:
I'm referring to the `created_at` attribute returned from Horizon's `GET /transactions/:id` endpoint. This value is the close time of the ledger the transaction was included in. This gives us a source of trust to use so that there is never a dispute about whether the payment was made prior to the quote's expiration.